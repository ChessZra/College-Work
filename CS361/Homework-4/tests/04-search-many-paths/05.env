PATH=/path1:/path2:/path3:/path4:/path5:/path6:/path7:/path8:/path9:/path10:/path11:/path12:/path13:/path14:/path15:/path16:/path17:/path18:/path19:/path20:/path21:/path22:/path23:/path24:/path25:/path26:/path27:/path28:/path29:/path30:/path31:/path32:/path33:/path34:/path35:/path36:/path37:/path38:/path39:/path40:/path41:/path42:/path43:/path44:/path45:/path46:/path47:/path48:/path49:/path50:/path51:/path52:/path53:/path54:/path55:/path56:/path57:/path58:/path59:/path60:/path61:/path62:/path63:/path64:/usr/bin