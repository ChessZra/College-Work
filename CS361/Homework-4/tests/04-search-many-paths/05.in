basename -a /path/one /pathother/two
