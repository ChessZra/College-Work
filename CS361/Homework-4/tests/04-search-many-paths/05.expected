cs361sh> one
two
cs361sh> 
